.github/workflows/buildcheck.yml

```diff
@@ -28,7 +29,10 @@ jobs:
         run: make travis_targets
 
       - name: Test optional targets. NOTE - always succeeds
-        run: make -k travis_optional || true
+        run: |
+          sudo apt-get install -y pandoc  # For csv2pdf
+          make -k travis_optional || true
+          ls -al vss-tools/contrib *.pdf
```

Comment:
Noticed that the last check failed due to the "frozen" error that hopefully is fixed with the last commit in vss-tools.

But the reason for the failed check was that `ls` did not find any pdf file. If we want that part to always pass we should add `true`like below

`ls -al vss-tools/contrib *.pdf || true`